Automattic/WP-Job-Manager
# Use global for select2 args and set width to 100%

This fixes an issue with window resizing and renames the `job_manager_select2_multiselect_args` variable to more generic `job_manager_select2_args`.

To fix the issue with window resizing, it passes a more opinionated 100% width. Seems to look okay in the themes I tested with.

### Testing Instructions
- On multiple themes, check:
   - `[jobs]` filter with multiselect and normal select for the job categories.
   - `[submit_job_listing]` page with multiselect fields.

These should all be `select2` unless `chosen` has already loaded from the theme or the theme is explicitly blocking it.
